packages/fuchsia_ctl/bin/main.dart

```diff
@@ -50,8 +51,12 @@ Future<void> main(List<String> args) async {
         help: 'The command to run on the device. '
             'If specified, --interactive is ignored.')
     ..addOption('identity-file',
-        defaultsTo: '.ssh/pkey', help: 'The key to use when SSHing.');
+        defaultsTo: '.ssh/pkey', help: 'The key to use when SSHing.')
+    ..addOption('timeout-seconds',
+        defaultsTo: '120', help: 'Ssh command timeout in secs.');
   parser.addCommand('pave')
+    ..addOption('pubkey',
```

Comment:
nit: we're not using abbreviations anywhere else for the full command name, just call this `public-key`.

Reply:
done